src/app/Events/EventTemplates.tsx

```diff
@@ -373,8 +374,12 @@ export const EventTemplates = () => {
     </>
   );
 
+  const authRetry = React.useCallback(() => {
+    context.target.setAuthRetry();
+  }, [context.target, context.target.setAuthRetry]);
+
   if (errorMessage != '') {
-    return <ErrorView message={errorMessage} title={'Fail to retrieve event templates'} />;
+    return <ErrorView title={'Fail to retrieve event templates'} message={errorMessage} retry={authRetry} />;
```

Comment:
Maybe change to either "Failed to retrieve event templates" and change the other one to match with it, or change to "Error retrieving event templates" so that this one matches with the other error message.

Reply:
Good idea! I will fix that!